e2e/integration_tests/data_flow_test.go

```diff
@@ -584,7 +566,7 @@ func Test_RetrieveCapabilityStatements(t *testing.T) {
 		"https://webproxy.comhs.org/FHIR/api/FHIR/DSTU2/",
 		"https://rwebproxy.elcaminohospital.org/FHIR/api/FHIR/DSTU2/",
 		"https://lmcrcs.lexmed.com/FHIR/api/FHIR/DSTU2/",
-		"https://proxy.cfmedicalcenter.com/FHIRProxyPRD/api/FHIR/DSTU2/"}
+		"https://dev-eagle.azurehealthcareapis.com/"}
```

Comment:
I feel like I'm missing something here since it looks like there are only 5 endpoints that were deleted from `TestEndpointSources.json` to make `TestEndpointSources_1.json` and dev-eagle and eloh-mapilive aren't there

Reply:
Good eye, this was a find/replace error that I did not mean to commit.